doc/sphinxman/source/build_planning.rst

```diff
@@ -301,6 +301,29 @@ any/all |PSIfour| builds detect that installation at compile-time.
 * :ref:`cmake:libint`
 
 
+.. _`faq:condamaxameri`:
+
+How to get high angular momentum integrals from conda
+-----------------------------------------------------
+
+To switch from the default libint package to the really large high AM
+package, do the below. The channel/subchannel(s) containing the `am8`
+metapackage and the high-AM libint package must be supplied (or in
+.condarc). ::
+
+    conda install am8 -c psi4
+
```

Comment:
Thanks. This is my first successful use of conda features, but I figured I'd better master them b/c `avx` and so forth are going to want them.